### PR TITLE
Fix Grammar and remove errors

### PR DESCRIPTION
**Pull request by iShashank04**
**Fixes**: #

🚨 Please review the [guidelines for contributing](CONTRIBUTING.md) and our [code of conduct](../CODE_OF_CONDUCT.md) to this repository. 🚨
**Please complete these steps and check these boxes (by putting an x inside the brackets) before filing your PR:**

- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.

#### Proposed changes:

-
-
-

👍 Thank you!

### CODE REVIEW

**Review by bitcoinfinancier — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Examples of unacceptable behavior by participants include:
 * The use of sexualized language or imagery and unwelcome sexual attention or advances
 * Trolling, insulting/derogatory comments, and personal or political attacks
 * Public or private harassment
-* Publishing others' private information, such as a physical or electronic address, without explicit permission
+* Publishing other's private information, such as a physical or electronic address, without explicit permission
```
Comment (grejojoby) [inline CODE_OF_CONDUCT.md:22]: This is not grammatically correct. others' is rightly used already. 

others' stands for a plural usage, which is correct already.

File: README.md [hunk 2]
```diff
@@ -288,13 +288,13 @@ Visualize and generate automatically our social meta tags with [Meta Tags](https
 
 > * 🛠 [W3C validator](https://validator.w3.org/)
 
-* [ ] **HTML Lint:** ![High][high_img] I use tools to help me analyze any issues I could have on my HTML code.
+* [ ] **HTML Lint:** ![High][high_img] I use tools to help me analyze any issues I could have with my HTML code.
```
Comment (grejojoby) [inline README.md:291]: You should use "in" instead of "with".

Comment (grejojoby) [inline README.md:291]:
```suggestion
* [ ] **HTML Lint:** ![High][high_img] I use tools to help me analyze any issues I could have in my HTML code.
```

**Review by grejojoby — CHANGES_REQUESTED** [review]
Other changes seems fine.

**Pull request closed without merging**